Bugsnag/Payload/BugsnagEvent.m

```diff
@@ -339,6 +339,30 @@ - (instancetype)initWithKSCrashData:(NSDictionary *)event {
     NSDictionary *error = [event valueForKeyPath:@"crash.error"];
     NSString *errorType = error[BSGKeyType];
 
+    // Data coming from KSCrash will have either user.handledCount = 0 and user.unhandledCount > 0, or the opposite
```

Comment:
This comment is misleading - the values of `handledCount` and `handledCount` come from the session tracker and we cannot assume that one of them will be 0.

For example, prior to the crash there may have been multiple non-fatal errors reported, and they could have been counted as handled or unhandled depending on whether the developer overrode the unhandled parameter.